src/control/dmg/main.go

```diff
@@ -25,17 +25,22 @@ package main
 
 import (
 	"fmt"
-	"os"
-	"strings"
 
 	"github.com/daos-stack/daos/src/control/client"
 	"github.com/daos-stack/daos/src/control/log"
+	"github.com/daos-stack/daos/src/control/security"
+	"google.golang.org/grpc/credentials"
+
+	"os"
```

Comment:
Why did these imports get moved? As they're stdlib imports, they should be grouped with fmt.

Reply:
I'd imagine it was vs.code that move them. I just saved and this is what it gave me. I can explicitly order them.

Reply:
A common convention for imports is to group stdlib imports first, then 3rd party imports, then internal imports. They then get sorted within their groups. This convention makes it easier for a reader to quickly see what's being brought in.